### ISSUE TITLE
[CONTENT] Add wrong/debugging examples to slides

### ISSUE THREAD

**Issue by Felienne** [hedyorg/hedy#4784]
In [this issue](https://github.com/hedyorg/hedy/issues/3967) a great suggestion is being made: adding wrong examples to slides and have learners figure out what is wrong. 

I think that is a lovely idea!

Pasted here: (cause the comment is unreasonably long :) )

## A social solution

A possible solution that wouldn't involve needing to modify error messages would be: Introduce exercises in hedy that intentionally give students error messages and walk them through fixing them. Ask them socratic method questions about why the computer might be giving you that "weird" output with hints. #4043 could possibly help here if it's not a good fit for the existing hedy interface.

It would actually be really nice to have each section contain some some working and non-working examples of the syntax.

I'm imagining something like:

- Teacher shows:

```
print 'Hello world
```

Teacher asks: What do you think happens when you run this code?
Students: ...
Teacher: Let's try it.

```
Be careful. If you `ask` or `print` something, the text should start and finish with a quotation mark.
You forgot that for the text 'Hello world.
```

Teacher: That's not what we expected, what went wrong?
Students:
Teacher: Here's how we fix it...

Teacher: What about this code:

```
animals is dog cat kangaroo
add penguin to animals
print animals at random
```
Student: ...
Teacher: Here's the output:

```
You cannot use add to with animals because it is text. Try changing animals to a list.
```

Teacher: Why does it think that animals is a string? Did we forget something? How do we fix it?

Then:

Teacher: How about this code:

```
print 'Hello, world
```

Teacher: It says there's a problem with the comma. What if we remove it, what do you think will happen, do you think that will fix the program? No? Well...let's try anyway to see what happens.

**Comment by MarleenGilsing** [comment]
I love this idea, but I see one small problem: As far as I know I can only use startcodes in the slides at the moment. I would need to be able to add more codes than just startcodes to the slides., before I can make this.

**Comment by LegoPink** [comment]
In the slides for Level 2, I found one on debugging, great!
Only, I'm teaching dutch 9 year olds, and I teach them in dutch. The debugging slide is in English.

Also, the last slide in Level 2 says: 
"Veel plezier met de avonturen in level 3!"

**Comment by Felienne** [comment]
> In the slides for Level 2, I found one on debugging, great! Only, I'm teaching dutch 9 year olds, and I teach them in dutch. The debugging slide is in English.

Yes, they still need to be translated. If you want to.... you could do that of course. Otherwise we will get around to it in a few weeks certainly!

> Also, the last slide in Level 2 says: "Veel plezier met de avonturen in level 3!"

Good catch! That is an easy fix, I wil update it right away.

**Comment by Felienne** [comment]
> Good catch! That is an easy fix, I wil update it right away.

Done in #4975

**Comment by MarleenGilsing** [comment]
@LegoPink, thanks for reminding me about the missing translations. We have made quite a lot of new content lately, so I've made a seperate issue for my or Sabina's to-do list that we check everything for missing DUtch translations.

**Comment by MarleenGilsing** [comment]
Fixed by #5435
